core/io/tcp_server.h

```diff
@@ -44,11 +44,16 @@ class TCP_Server : public Reference {
 		MAX_PENDING_CONNECTIONS = 8
 	};
 
+	uint16_t local_port = 0;
+	IP_Address local_address;
+
 	Ref<NetSocket> _sock;
 	static void _bind_methods();
 
 public:
 	Error listen(uint16_t p_port, const IP_Address &p_bind_address = IP_Address("*"));
+	Error get_address(IP_Address &local_address, uint16_t &local_port) const;
+	int get_local_port() const;
```

Comment:
Should return uint16_t instead of int

Reply:
Please see my [previous answer](https://github.com/godotengine/godot/pull/43174/files/4548b790985cdd0adbb6d5fb2f7498584ecc2f55#r528388372) to a similar comment.
Again, I see no problem in changing this to `uint16_t`, I just want to double check.